app/components/ToDos.js

```diff
@@ -12,53 +12,38 @@ import { CheckBox, Icon } from "react-native-elements";
 import { connect } from "react-redux";
 
 import TodosQuery from "../redux/queries/todos"
-import { addTodo, completeTodo } from "../redux/actions/todos";
+import { addTodo, completeTodo, deleteTodo } from "../redux/actions/todos";
 
 //type Props = {};
 class ToDos extends Component<Props> {
 
   componentDidMount(){
-    this.props.addTodo({
+    const { addTodo } = this.props; 
+    addTodo({
       id: 1,
       toDo: "Build a Todo App",
       completed: true
     });
-    this.props.addTodo({
+    addTodo({
       id: 2,
       toDo: "Do presentation",
       completed: false
     });
-    this.props.addTodo({
+    addTodo({
```

Comment:
Looking at this function in its entirety, you could reduce some of the complexity.

```javascript
const todos = [
  {
    id: 1,
    toDo: "Build a Todo App",
    completed: true,
  },
  // …
];

todos.forEach(todo => this.props.addTodo(todo));
```

This way you have one line of actual code and the rest is data.